websites/P/PresenceDB/dist/metadata.json

```
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "vasc",
+    "id": "346977366569910274"
+  },
+  "service": "PresenceDB",
+  "description": {
+    "en": "Tracks and saves the playtime of your Discord activities.",
+    "de": "Verfolgt und speichert die Spielzeit Ihrer Discord Aktivitäten."
+  },
+  "url": ["www.presencedb.com", "preview.presencedb.com"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/tXfdJDT.png",
+  "thumbnail": "https://i.imgur.com/P20KlQh.png",
+  "color": "#fff",
+  "tags": ["presencedb", "database"],
+  "category": "other",
+  "regExp": "(www|preview)[.]presencedb[.]com[/]([a-z]*)",
```

```suggestion
  "regExp": "(www|preview)[.]presencedb[.]com[/]",
```
You shouldn't need that (just double check that it still works)